packages/containers/src/HomeListView/HomeListView.container.js

```diff
@@ -26,16 +26,7 @@ class HomeListView extends React.Component {
 	}
 
 	render() {
-		const { sidepanel, list, children, ...props } = this.props;
-		return (
-			<Component
-				sidepanel={sidepanel}
-				list={list}
-				{...props}
-			>
-				{children}
-			</Component>
-		);
+		return (<Component {...this.props} />);
```

Comment:
The bug #151 was here because we detructured the props and pass only `sidepanel` and `list`, thus the children was not used, not passed.
Here we pass everything